cookbooks/aws-parallelcluster-slurm/resources/slurm_dependencies/slurm_dependencies_alinux2023.rb

```diff
@@ -0,0 +1,41 @@
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+provides :slurm_dependencies, platform: 'amazon' do |node|
+  node['platform_version'].to_i == 2023
+end
+
+use 'partial/_slurm_dependencies_common'
+
+def dependencies
+  %w(json-c-devel perl perl-Switch lua-devel dbus-devel)
+end
+
+def unsupported_dependencies
+  # Using `sudo dnf supportinfo --pkg <PACKAGE_NAME>` to find if packages are available
+  %w(http-parser-devel)
+  # Tried replacing http-parser with below Packages ( doesnt work )
```

Comment:
IS this verification true of just a copy paste.?What would be the advantage of using the commented out packages?

Reply:
This was me trying to keep an array of dependencies which I couldnt find in AL2023 repo's. 
I wanted to track these dependencies in the code rather than removing them as later if they are supported we dont have to scramble to compare the code with different OS to know which dependency is missing.